javascript/javascript1/week1/dog_age.js

```diff
@@ -0,0 +1,14 @@
+const dogYearOfBirth = 1957;
+const dogYearFuture = 2027;
+console.log(dogYearFuture - dogYearOfBirth);
+const dogYear = 60;
+const dog_age= dogYearFuture - dogYearOfBirth;
+if (dog_age <=dogYear) { 
+ shouldShowResultInDogYears = true;
+console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture);
+ }
+else{
+shouldShowResultInDogYears = false;
```

Comment:
Remember to always declare your variables. If you plan on re-assigning this one you could declare 
```js
let shouldShowResultInDogYears = true
```

at the top of file and re-assign it to false when needed.